content/applications/general/developer_mode.rst

```diff
@@ -4,77 +4,86 @@
 
-The command palette tool has a command to activate the debug mode: open it with
-the keyboard shortcut `ctrl+k`, then type `debug`: a command will show up to
+In Odoo, the command palette tool has a command to activate the debug mode. First, open the command
+palette tool with the keyboard shortcut `ctrl+k` and then type `debug`. A command will show up to
 activate the debug mode.
 
-.. image:: developer_mode/command_palette.png
+.. image:: developer_mode/command-palette.png
    :align: center
-   :alt: Command palette with debug command
+   :alt: Command palette with debug command.
 
 Activate through the URL
 ========================
 
-In the URL, add ``?debug=1`` or ``?debug=true`` after *web*. To deactivate the
-debug mode, add `?debug=0` instead.
+The debug mode can also be activated by adding an extra query string to the database's URL. In the
+URL, add ``?debug=1`` or ``?debug=true`` after *web*. To deactivate the debug mode, add `?debug=0`
+instead.
 
 .. image:: developer_mode/url.png
    :align: center
-   :alt: Overview of an url with the debug mode command added in Odoo
+   :alt: Overview of a URL with the debug mode command added.
 
 .. tip::
-   Additional modes are available for developers: `?debug=assets` enables the
-   :ref:`assets mode <frontend/framework/assets_debug_mode>`, and `?debug=tests` enables
-   the :ref:`tests mode <frontend/framework/tests_debug_mode>`.
+   Additional modes are available for developers: `?debug=assets` enables the :ref:`assets mode
+   <frontend/framework/assets_debug_mode>`, and `?debug=tests` enables the :ref:`tests mode
+   <frontend/framework/tests_debug_mode>`.
 
 .. _developer-mode/mode-tools:
 
-Locate the mode tools
-=====================
-
-The Developer mode tools can be accessed from the *Open Developer Tools* button,
-located on the header of your pages. This menu contains additional tools that
-are useful to understand or edit technical data, such as the views or the actions.
-It contains some useful menu items such as:
+Locate the developer mode tools
+===============================
 
-- edit action
-- manage filters
-- edit the current view
-- see the `fields view get`
-- and much more.
+Once developer mode is activated, the developer mode tools can be accessed from the *Open Developer
+Tools* button, the bug icon located in the header of the Odoo database.
 
-.. image:: developer_mode/button_location.png
+.. image:: developer_mode/button-location.png
    :align: center
-   :alt: Overview of a console page and the debug icon being shown in Odoo
+   :alt: Overview of a console page and the debug icon being shown in Odoo.
+
+This menu contains additional tools that are useful to understand or edit technical data, such as
+the views or actions of a page. On a page that has filters, actions, and view options, the
+developer mode tools contain some useful menu items such as:
+
+- :guilabel:`Edit Action`
+- :guilabel:`Manage Filters`
+- Edit the current view
```

Comment:
Since the button values are dynamic and we're not using a `:guilabel:` here I would add in example views to help the reader make the connection when looking for this item on the debug menu.

```suggestion
- Edit the current view (e.g. Kanban, List, Graph, etc.)
```